frontend/src/metabase/home/components/Onboarding/Onboarding.tsx

```diff
@@ -103,43 +105,54 @@ export const Onboarding = () => {
     databaseId: lastUsedDatabaseId || undefined,
   });
 
-  const sendMessage = (command: string, value: IframeKeys) => {
-    const iframeRef = iframeRefs[value];
-    if (iframeRef.current) {
-      iframeRef.current?.contentWindow?.postMessage(
-        JSON.stringify({
-          event: "command",
-          func: command,
-          args: [],
-        }),
-        "*",
-      );
-    }
-  };
-
   const [lastItemOpened, setLastItemOpened] = useTempStorage(
     "last-opened-onboarding-checklist-item",
   );
 
-  useEffect(() => {
-    if (lastItemOpened && isValidIframeKey(lastItemOpened)) {
-      iframeRefs[lastItemOpened]?.current?.scrollIntoView({
-        behavior: "smooth",
-        block: "center",
-      });
+  const scrollElementIntoView = (element?: HTMLDivElement | null) => {
+    if (!element) {
+      return;
     }
-  }, [iframeRefs, lastItemOpened, isValidIframeKey]);
+    element.scrollIntoView({
+      behavior: "smooth",
+      block: "center",
+    });
+  };
 
-  const stopVideo = (key: IframeKeys) => sendMessage("stopVideo", key);
+  // Scroll the last opened item into view when the user navigates back go this page
+  useEffect(() => {
+    if (isValidItemKey(lastItemOpened)) {
+      const item = itemRefs[lastItemOpened].current;
+      scrollElementIntoView(item);
+    }
+  }, [itemRefs, lastItemOpened, isValidItemKey]);
+
+  const stopVideo = (iframe: HTMLIFrameElement) =>
```

Comment:
Can this live outside the component definition? There are no variables being closed over AFAICT.